vm/ubpf_inst_cnt.c

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2021 Eideticom, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#define _GNU_SOURCE
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <stdarg.h>
+#include <inttypes.h>
+#include <unistd.h>
+#include <sys/syscall.h>
+#include <sys/ioctl.h>
+#include <sys/mman.h>
+#include "ubpf_int.h"
+#ifdef __linux__
```

Comment:
Nit: would be better to have a blank line here. The #ifdef isn't very visible butted up against the includes.

Reply:
Also, where'd all these headers come from? Very few of them actually look used in here.... 

Best to put "ubpf_int.h" at the top as is best practice in C.

Reply:
@lsgunth I addressed this.